packages/shared/src/components/search/SearchHistory.tsx

```diff
@@ -19,12 +21,19 @@ export function SearchHistory({
   className,
   title,
 }: SearchHistoryProps): ReactElement {
+  const { trackEvent } = useContext(AnalyticsContext);
   const {
     result: { isLoading },
     nodes,
     infiniteScrollRef,
   } = useSearchHistory();
 
+  useEffect(() => {
+    trackEvent({ event_name: AnalyticsEvent.OpenSearchHistory });
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
I think we need to change this one to trigger once clicking the search history tab.
It might trigger multiple times that way.